pom.xml

```diff
@@ -44,7 +44,7 @@
 
         <!--Spring Versions-->
         <spring-framework.version>5.3.23</spring-framework.version>
-        <spring-security.version>6.1.0</spring-security.version>
+        <spring-security.version>6.0.2</spring-security.version>
```

Comment:
do we have to upgrade spring security as the other spring dep versions are below v6